Sources/sentry/include/SentryApp.h

```diff
@@ -5,4 +5,28 @@
 
 #include <sentry.h>
 
-#endif
\ No newline at end of file
+#ifdef _WIN32
+#include <atomic>
+
+// The type of swift::swift_getFatalErrorMessageBuffer
+using swift_getFatalErrorMessageBuffer_t = std::atomic<const char *> *(*)();
+
+inline void *getFatalErrorMessageHandle() {
+  HMODULE hSwiftCore = GetModuleHandleA("swiftCore.dll");
+  if (!hSwiftCore) { return nullptr; }
+  swift_getFatalErrorMessageBuffer_t pGetBuf =
```

Comment:
I'm not sure I see the value of `swift_getFatalErrorMessageBuffer_t`. You only need to reference it once - for the `reinterpret_cast` "type specifier", as the VarDecl can use `auto` storage. We do not get any sharing for the evaluation, so this is truly a single use type definition.

Reply:
Done